src/improvements/tasks/sep/005-opcm-upgrade-v300/VALIDATION.md

```diff
@@ -0,0 +1,527 @@
+  
+- **Summary:**           ERC-1967 implementation slot
+  - **Detail:**            Standard slot for storing the implementation address in a proxy contract that follows the ERC-1967 standard.
+  
+**TODO: Insert links for this state change.**
+
+  ---
+  
+### `[0x33f60714bbd74d62b66d79213c348614de51901c](https://github.com/ethereum-optimism/superchain-registry/blob/00208555c3c356d6596feedb619da989de478ed7/superchain/configs/sepolia/ink.toml#L56-L56) (L1StandardBridge - Ink Sepolia)`
+  
+#### Decoded State Change: 7
+  - **Contract:**          `L1StandardBridge`
+  - **Chain ID:**          `763373`
+  
+- **Key:**          `0x360894a13ba1a3210667c828492db98dca3e2076cc3735a920a3ca505d382bbc`
+  - **Decoded Kind:**      `address`
+  - **Before:** `0x78972E88Ab8BBB517a36cAea23b931BAB58AD3c6`
+  - **After:** `[0x0b09ba359A106C9ea3b181CBc5F394570c7d2a7A](https://github.com/ethereum-optimism/superchain-registry/blob/00208555c3c356d6596feedb619da989de478ed7/validation/standard/standard-versions-sepolia.toml#L20-L20)`
+  
+- **Summary:**           ERC-1967 implementation slot
+  - **Detail:**            Standard slot for storing the implementation address in a proxy contract that follows the ERC-1967 standard.
+  
+  ---
+  
+### `[0x3454f9df5e750f1383e58c1cb001401e7a4f3197](https://github.com/ethereum-optimism/superchain-registry/blob/00208555c3c356d6596feedb619da989de478ed7/superchain/configs/sepolia/ink.toml#L53-L53) (AddressManager - Ink Sepolia)`
+  
+#### Decoded State Change: 8
+  - **Contract:**          `AddressManager`
+  - **Chain ID:**          `763373`
+  
+- **Key:**          `0x515216935740e67dfdda5cf8e248ea32b3277787818ab59153061ac875c9385e`
+  - **Before:**     `0x0000000000000000000000003ea6084748ed1b2a9b5d4426181f1ad8c93f6231`
+  - **After:**     `0x0000000000000000000000005d5a095665886119693f0b41d8dfee78da033e8b`
+  
+- **Summary:**  The name `OVM_L1CrossDomainMessenger` is set to the address of the new 'op-contracts/v3.0.0-rc.2' L1CrossDomainMessenger at [0x5d5a095665886119693f0b41d8dfee78da033e8b](https://github.com/ethereum-optimism/superchain-registry/blob/1a5d7a208cea9b0ea175df1fe71bdc4da7f4c04c/validation/standard/standard-versions-sepolia.toml#L18).
+  - **Detail:** This key is complicated to compute, so instead we attest to correctness of the key by
+                     verifying that the "Before" value currently exists in that slot, as explained below.
+                     **Before** address matches both of the following cast calls:
+                      1. What is returned by calling `AddressManager.getAddress()`:
+                       - cast call 0x3454F9df5E750F1383e58c1CB001401e7A4f3197 'getAddress(string)(address)' 'OVM_L1CrossDomainMessenger' --rpc-url sepolia
+                      2. What is currently stored at the key:
+                       - cast storage 0x3454F9df5E750F1383e58c1CB001401e7A4f3197 0x515216935740e67dfdda5cf8e248ea32b3277787818ab59153061ac875c9385e --rpc-url sepolia
```

Comment:
@blmalone WDYT of this change?

```suggestion
  - **Detail:** **THIS WAS CAREFULLY VERIFIED BY RUNBOOK REVIEWERS AND NEED NOT BE CHECKED BY SIGNERS.**
    This key is complicated to compute, so instead we attest to correctness of the key by
    verifying that the "Before" value currently exists in that slot, as explained below.
    **Before** address matches the following cast call to `AddressManager.getAddress()`:
      - `cast call 0x3454F9df5E750F1383e58c1CB001401e7A4f3197 'getAddress(string)(address)' 'OVM_L1CrossDomainMessenger' --rpc-url sepolia`
      - returns: `0x3eA6084748ED1b2A9B5D4426181F1ad8C93F6231`
```

Reply:
Yes, I'm all for giving signers explicit messages so that it makes their lives easier. Lets add this in.